randomActivityGen.py

```diff
@@ -240,7 +240,9 @@ def main():
         setup_schools(net, stats, int(args["--schools.count"]))
 
     # Write statistics back
-    stats.write(args["--output-file"])
+    output_file = args["--output-file"]
+    os.makedirs(os.path.dirname(output_file), exist_ok=True)
```

Comment:
From my experience programs doesn't normally create the parent directories to the output location. It would be normal to create a `output` directory, but not the parent of it

Reply:
Okay. We will do it like that then.